instance/github.py

```diff
@@ -157,8 +164,39 @@ def __init__(self, number, fork_name, branch_name, title, username, body=''):
         self.body = body
 
     @property
+    def settings(self):
+        """
+        Settings contained in the PR body
+        """
+        return yaml.load(get_settings_from_pr_body(self.body)) or {}
+
+    @property
+    def database_url(self):
+        """
+        An optional database url, from the DATABASE_URL setting in the PR body
+        """
+        return self.settings.get(self.SETTINGS['DATABASE_URL'])
+
+    @property
+    def mongo_url(self):
```

Comment:
I think you would be good with these 3 - for elasticsearch we currently don't enable it on the sandboxes, so you can ignore it for now.